source/common/http/http2/codec_impl.h

```diff
@@ -596,7 +597,7 @@ class ConnectionImpl : public virtual Connection,
   void sendSettingsHelperOld(const envoy::config::core::v3::Http2ProtocolOptions& http2_options,
                              bool disable_push);
   // Callback triggered when the peer's SETTINGS frame is received.
-  virtual void onSettings(const nghttp2_settings& settings) {
+  virtual void onSettings(const std::vector<http2::adapter::Http2Setting>& settings) {
```

Comment:
super optional, here and elsewhere: Could do absl::Span.

Reply:
Done, thank you.